src/ipa/rpi/controller/rpi/awb.cpp

```diff
@@ -476,11 +398,7 @@ void Awb::prepareStats()
 	 * LSC has already been applied to the stats in this pipeline, so stop
 	 * any LSC compensation.  We also ignore config_.fast in this version.
 	 */
-	const double biasCtR = config_.bayes ? config_.ctR.eval(config_.biasCT) : 0;
-	const double biasCtB = config_.bayes ? config_.ctB.eval(config_.biasCT) : 0;
-	generateStats(zones_, statistics_, config_.minPixels,
-		      config_.minG, getGlobalMetadata(),
-		      config_.biasProportion, biasCtR, biasCtB);
+	generateStats(zones_, statistics_, 0.0, 0.0, getGlobalMetadata(), 0.0, 0.0, 0.0);
```

Comment:
AwbBayes seems to have its own prepareStats, is that right? So is this the AwbNN version? Might be better to leave it pure virtual in the base class and define it in awb_nn.cpp.